### DISCUSSION TITLE
Increasing capacity with SEI submodel

### DISCUSSION THREAD

**Discussion by aminacoric** [pybamm-team/PyBaMM#2075] [Q&A]
Hi, I have a question regarding the capacity plot using plot_summary_variables().

I did a simulation using SPM with reaction limited SEI submodel during 1200 CC cycles (Charge to upper cut-off voltage and discharge to lower cut-off voltage). The result in the _capacity_ plot using plot_summary_variables() shows that the capacity is increasing instead of, what I expected, to be decreasing because of the SEI growth. But the plot for _measured capacity_ is decreasing. I wonder what the difference is between these two and how to interpret the increasing capacity plot? 

![image](https://user-images.githubusercontent.com/103030682/169991860-ad966347-8fc5-4e50-b136-ccda460b1381.png)

**Comment by brosaplanella** [comment]
That's weird... Can you share a minimum working example of this?
